test/spec/runners/ssz_static.ex

```diff
@@ -0,0 +1,55 @@
+defmodule SSZStaticTestRunner do
+  use ExUnit.CaseTemplate
+  alias LambdaEthereumConsensus.Ssz
+
+  @moduledoc """
+  Runner for SSZ test cases. `run_test_case/1` is the main entrypoint.
+  """
+
+  @doc """
+  Returns true if the given testcase should be skipped
+  """
+  def skip?(testcase) do
+    # add SSZ test case skipping here
+    testcase.handler not in ["Checkpoint", "Fork", "ForkData"]
+  end
+
+  @doc """
+  Runs the given test case.
+  """
+  def run_test_case(testcase = %SpecTestCase{}) do
+    case_dir = SpecTestCase.dir(testcase)
+
+    compressed = File.read!(case_dir <> "/serialized.ssz_snappy")
+    assert {:ok, decompressed} = :snappyer.decompress(compressed)
+
+    expected =
+      YamlElixir.read_from_file!(case_dir <> "/value.yaml")
+      |> Stream.map(&parse_yaml/1)
+      |> Map.new()
+
+    expected_root = YamlElixir.read_from_file!(case_dir <> "/roots.yaml")
+
+    assert_ssz(testcase, decompressed, expected, expected_root)
```

Comment:
Can we include the Elixir struct here somewhere?

like: decompressed = %Checkpoint{}

Reply:
So that we ensure that the Elixir structs are respecting the schema.

Reply:
I don't know if this is what you mean, but I intend to add the Elixir-side structs and integrate that with the SSZ lib and the test runner in another PR.

Reply:
sounds good